source/operations/install-deploy-manage/upgrade-minio-operator.rst

```diff
@@ -204,14 +113,15 @@ Upgrade Operator to |operator-version-stable|
 
             kubectl get pod -l 'name=minio-operator' -n minio-operator -o json | jq '.items[0].spec.containers'
 
-      #. *(Optional)* Connect to the Operator Console
+      .. important::
 
-         .. include:: /includes/common/common-k8s-connect-operator-console-no-plugin.rst
+         Operator 6.0.0 deprecates the MinIO Operator Console and removes the related resources from the MinIO Operator CRD.
 
-      #. Retrieve the Operator Console JWT for login
-
-	 .. include:: /includes/common/common-k8s-operator-console-jwt.rst
+         MinIO does not remove the old Console services, pods, or deployments as per Kubernetes standard procedure.
+         You can remove these resources at your own discretion.
+         MinIO is no longer developing nor supporting the Console, and as of such advises against long-term reliance on the Console for Tenant management.
```

Comment:
Might be worth the additional emphasis here
```suggestion
         MinIO is no longer developing nor supporting Operator Console, and as of such advises against long-term reliance on the Console for Tenant management.
```

Reply:
Also `long-term reliance on the Console for Tenant management` -> `long-term reliance on this Console for Tenant management`. Make sure it's clear which Console we are deprecating.